atlassian_jwt_auth/contrib/django_app/decorators.py

```diff
@@ -0,0 +1,41 @@
+from functools import wraps
+from django.http import HttpResponse, HttpResponseForbidden
+from django.conf import settings
+
+import jwt
+
+from atlassian_jwt_auth.contrib.django_app.auth import ASAPAuthBackend
+
+
+class HttpResponseNotAuthorized(HttpResponse):
```

Comment:
I did find that peculiar as well. I suppose they figure that generally someone writing a Django application and using stock backends will not need to raise a 401 since the framework will do it for them?